crates/gpui/src/input.rs

```diff
@@ -15,6 +15,10 @@ pub trait ViewInputHandler: 'static + Sized {
     /// See [`InputHandler::selected_text_range`] for details
     fn selected_text_range(&mut self, cx: &mut ViewContext<Self>) -> Option<Range<usize>>;
 
+    #[cfg(target_os = "linux")]
+    /// See [`InputHandler::selection_position`] for details
+    fn selection_position(&mut self, cx: &mut ViewContext<Self>) -> usize;
```

Comment:
It would be better if we could use the 'selected_text_range' API for this, as it's already the same thing. Maybe we need a new return type that allows to specify which end is the 'head'?